docs/guides/admin/docs/releasenotes.md

```diff
@@ -24,6 +24,9 @@ New Features
 - Display responsible person for workflows
 - Allow the Ingest Service to make authenticated requests to other servers
 - Crop service
+- Some features are now plugins and will not anymore started, regardless of people using them or not:
```

Comment:
Suggest rephrase to:

These features are now plugins, and will not be started by default unless enabled in `etc/org.apache.karaf.features.cfg`

Reply:
Rephrased although it's just a suggestion for the release managers anyway.